CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+1.0.2 - 2017-02-22
+------------------
+
+* The factory now returns the modified instance of tv4.
```

Comment:
Can you a bit more explicit? you are making this work nicer with Dizzy. Maybe put a comment about that.